sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -369,6 +371,40 @@ def _read_eeprom_devid(self, port_num, devid, offset, num_bytes = 256):
 
         return eeprom_raw
 
+    def _write_eeprom_specific_bytes(self, sysfsfile_eeprom, offset, num_bytes, write_buffer):
+
```

Comment:
Suggest deleting this blank line to match the style of the rest of the file

Reply:
fixed